src/requirements/data/templates/ece.ts

```diff
@@ -0,0 +1,86 @@
+import { Template } from '@/requirements/types';
+
+const eceTemplate: Template = [
+  {
+    reqGroup: 'UNI',
+    name: 'Physical Education',
+    placeholderSemesters: [1, 2],
+  },
+  {
+    reqGroup: 'EN',
+    name: 'Mathematics',
+    placeholderSemesters: [1, 2, 3, 4],
+  },
+  {
+    reqGroup: 'EN',
+    name: 'Physics',
+    placeholderSemesters: [2, 3, 4],
+  },
+  {
+    reqGroup: 'EN',
+    name: 'Chemistry',
+    placeholderSemesters: [1],
+  },
+  {
+    reqGroup: 'EN',
+    name: 'First-Year Writing Seminars',
+    placeholderSemesters: [1, 2],
+  },
+  {
+    reqGroup: 'EN',
+    name: 'Advanced Programming/Computer Engineering',
+    placeholderSemesters: [3],
+  },
+  {
+    reqGroup: 'EN',
+    name: 'ENGRD 2300',
+    placeholderSemesters: [3],
+  },
+  {
+    reqGroup: 'EN',
+    name: 'Core Courses',
+    placeholderSemesters: [3, 4],
+  },
+  {
+    reqGroup: 'EN',
+    name: 'Engineering Distribution',
+    placeholderSemesters: [4],
+  },
+  {
+    reqGroup: 'EN',
+    name: 'Liberal Studies: 6 courses',
+    placeholderSemesters: [3, 4, 5, 6, 7, 8],
+  },
+  {
+    reqGroup: 'EN',
+    name: 'Outside ECE Technical Electives',
+    placeholderSemesters: [5, 6, 7],
+  },
+  {
+    reqGroup: 'EN',
+    name: 'Culminating Design Experience (CDE)',
+    placeholderSemesters: [7],
+  },
+  {
+    reqGroup: 'EN',
+    name: 'Advisor Approved Electives',
+    placeholderSemesters: [3, 4],
+  },
+  {
+    reqGroup: 'EN',
+    name: 'Foundation Courses',
+    placeholderSemesters: [5, 6],
+  },
+  {
+    reqGroup: 'EN',
+    name: 'Upper-Level ECE Electives',
+    placeholderSemesters: [6],
+  },
+  {
+    reqGroup: 'EN',
+    name: 'Upper-Level ECE Electives: 4000 level',
+    placeholderSemesters: [7, 8],
+  },
+];
+
+export default eceTemplate;
```

Comment:
I think some requirements are missing from just templates
- Computing from engineering in Semester 2
- Intro to engineering in Semester 1

Additionally, I think some data is missing from the reqs file and templates. @einc do you agree with all of these, and if so, could you update them Aryan?
- Phys 1110 for students starting in Fall 2021 (not sure how we only would require for some students though).
- PHYS 2214 or 2218 in semester 4

Reply:
The current placeholderSemesters for Physics is [2, 3, 4], because I thought I would include PHYS 2214 along with the other 2 physics classes. Would this not work?

Reply:
I don't think so, because Physics is an Engineering-wide requirement, but the PHYS 2214/2218 requirement is specific to ECE. I think we would need a new PHYS 2214/2218 checker in the ECE file.

Reply:
Note: Not going to handle PHYS 1110 ourselves as this is a broader eng change that will be fixed by other devs